src/uuid.js

```diff
@@ -1,34 +1,22 @@
 /**
- * Source: [jed's gist]{@link https://gist.github.com/982883}.
+ * Source: [jed's gist's comment]{@link https://gist.github.com/jed/982883?permalink_comment_id=3223002#gistcomment-3223002}.
```

Comment:
Thanks @Mercy811. LGTM.

Can you please [add a new test](https://github.com/amplitude/Amplitude-JavaScript/blob/79585d5b84d814932783c97bae851ac361b548a5/test/uuid.js#L3) to do a sanity check for collisions?

Reply:
Thanks @justin-fiedler  for reviewing. I added a test to ensure two generated UUIDs are not equal.

Reply:
I think we should check more than two. 10000?

Reply:
I would do 10000 as I cannot come up a more reasonable number right now. We could increase it later if it's necessary. cc: @kelsonpw